src/app/entry/delete/dialog/delete-entry-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 mat-dialog-title>Are you sure you want to delete this Dockstore {{ term }}?</h1>
```

Comment:
I suggest specifying the name of the entry. If not here, because it may cause wrapping issues in the title, then at least on line 4.

I sometimes run into with the AWS Console UI, where it says "Are you sure you want to delete this <entity type>. I may not be 100% what they're referring to (the dialog may be covering, and I get paranoid it's referring to some crucial resource, so I dismiss the dialog to make sure I'm deleting the right thing. And then I bring up the dialog again, but even then, did I mis-click?

In your screenshot, you can see the selected/targeted notebook, but there's not guarantee it won't be covered up on a smaller screen or if we reorg things some day.